integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/hive/ProxyCatalogHiveIT.java

```diff
@@ -0,0 +1,252 @@
+
+  @BeforeAll
+  public static void startIntegrationTest() throws Exception {
+    originHadoopUser = System.getenv(HADOOP_USER_NAME);
+    setEnv(HADOOP_USER_NAME, null);
+
+    System.setProperty("user.name", "datastrato");
+
+    Map<String, String> configs = Maps.newHashMap();
+    configs.put(OAuthConfig.AUTHENTICATOR.getKey(), AuthenticatorType.SIMPLE.name().toLowerCase());
+    registerCustomConfigs(configs);
+    AbstractIT.startIntegrationTest();
+    containerSuite.startHiveContainer();
+    HIVE_METASTORE_URIS =
+        String.format(
+            "thrift://%s:%d",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HIVE_METASTORE_PORT);
+    HiveConf hiveConf = new HiveConf();
+    hiveConf.set(HiveConf.ConfVars.METASTOREURIS.varname, HIVE_METASTORE_URIS);
+
+    // Check if hive client can connect to hive metastore
+    hiveClientPool = new HiveClientPool(1, hiveConf);
+
+    Configuration conf = new Configuration();
+    conf.set(
+        "fs.defaultFS",
+        String.format(
+            "hdfs://%s:%d",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT));
+    hdfs = FileSystem.get(conf);
+    JettyServerConfig jettyServerConfig =
+        JettyServerConfig.fromConfig(serverConfig, WEBSERVER_CONF_PREFIX);
+
+    String uri = "http://" + jettyServerConfig.getHost() + ":" + jettyServerConfig.getHttpPort();
+    System.setProperty("user.name", "test");
+    anotherClient = GravitinoClient.builder(uri).withSimpleAuth().build();
+    createMetalake();
+    createCatalog();
+    loadCatalogWithAnotherClient();
+  }
+
+  @AfterAll
+  public static void stop() throws Exception {
+    setEnv(HADOOP_USER_NAME, originHadoopUser);
+    anotherClient.close();
+  }
+
+  @Test
+  public void testOperateSchema() throws Exception {
+    String schemaName = GravitinoITUtils.genRandomName(SCHEMA_PREFIX);
+    String anotherSchemaName = GravitinoITUtils.genRandomName(SCHEMA_PREFIX);
+    NameIdentifier ident = NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, schemaName);
+    NameIdentifier anotherIdent = NameIdentifier.of(METALAKE_NAME, CATALOG_NAME, anotherSchemaName);
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+    properties.put(
+        "location",
+        String.format(
+            "hdfs://%s:%d/user/hive/warehouse/%s.db",
+            containerSuite.getHiveContainer().getContainerIpAddress(),
+            HiveContainer.HDFS_DEFAULTFS_PORT,
+            schemaName.toLowerCase()));
+    String comment = "comment";
+
+    catalog.asSchemas().createSchema(ident, comment, properties);
+    Database db = hiveClientPool.run(client -> client.getDatabase(schemaName));
+    Assertions.assertEquals(EXPECT_USER, db.getOwnerName());
+    Assertions.assertEquals(
+        EXPECT_USER, hdfs.getFileStatus(new Path(db.getLocationUri())).getOwner());
```

Comment:
What is the default user for HDFS?

Reply:
datastrato is the default user for HDFS.